File: pre-contest/a_42/sol/sol.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv) {
    int n;
    cin >> n;
    
    if(n < 42)
        cout << "More" << endl;
    else if(n > 42)
        cout << "Less" << endl;
    else
        cout << "Bingo!" << endl;

    return 0;
}

File: pre-contest/a_42/sol/sol.java
import java.util.Scanner;

public class a_42{
	public static void main(String args[]) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		
		if(n < 42)
			System.out.println("More");
		else if(n > 42)
			System.out.println("Less");
		else
			System.out.println("Bingo!");
	}
}
